pymatviz/ptable.py

```diff
@@ -1619,28 +1640,28 @@ def ptable_scatters(
             types. Defaults to True.
         elem_type_legend_kwargs (dict): kwargs to plt.legend(), e.g. to
             set the legend title, use {"title": "Element Types"}.
-
-    TODO: allow colormap with 3rd data dimension
     """
     # Initialize periodic table plotter
-    plotter = PTableProjector(
+    projector = PTableProjector(
         data=data,
-        colormap=None,
+        colormap=colormap,
         plot_kwargs=plot_kwargs,  # type: ignore[arg-type]
         hide_f_block=hide_f_block,
         elem_type_colors=elem_type_colors,
     )
 
     # Call child plotter: Scatter
-    plotter.add_child_plots(
+    child_kwargs = {"cmap": projector.cmap, "norm": projector.norm}
```

Comment:
is the `child_kwargs` keyword of `ptable_scatters` left by accident? currently it's just being overwritten so whatever the user passes makes no difference

Reply:
Sorry! I must have made a mistake here, thanks for catching this and I would fix this now :)